splice/static/js/controllers/authoring.js

```diff
@@ -98,9 +98,73 @@ angular.module('spliceApp').controller('authoringController', function($controll
      * Send tiles to backend for publication.
      * Assumes data is correct.
      */
+    var cloneTile = function (tile) {
```

Comment:
a better name for this function could be: `cloneTileNoAsset`

Another possible way to clone, with slight additional memory usage is to:
```javascript
var cloneTileNoAsset = function (tile) {
  var copy = JSON.parse(JSON.stringify(tile));
  ["imageURI", "enhancedImageURI"].map(function (attr){
    copy[attr] = undefined;
  });
  return copy;
}
```